README.md

```diff
@@ -1,2 +1,99 @@
-# Module_1_Capstone
-Event Page
+# HTML/CSS & JavaScript - Microverse Assigment: Module 1 Capstone project - Event page
+
+# Disclaimer!
+
+This website is a design created by Cindy Shin in Behance.
+- [` https://www.behance.net/adagio07`](https://www.behance.net/adagio07)
+
```

Comment:
- [ ] Please, let's add this as an acknowledgment instead of a disclaimer. Create an acknowledgment section in the README and attribute it to the design author.

[OPTIONAL]

- [ ] I encourage you to use the README template provided by microverse. In so doing, you'll be providing only relevant information. It is unnecessary to repeat the project's instructions  verbatim or it could be re-worded in a README style.